expo-zustand-styled-components/src/components/PRAndIssueHeader/PRAndIssueHeader.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import PRAndIssueHeader from './PRAndIssueHeader';
+import { render } from '@testing-library/react-native';
+
+describe('User dropdown', () => {
+  it('should mount', () => {
+    const wrapper = render(<PRAndIssueHeader cardType="pr" openCount={10} closedCount={9} />);
+    expect(wrapper).toBeTruthy();
+    const option = wrapper.getByText('10');
```

Comment:
can we use the name of what you are looking out for here?
 ```suggestion
    const openCount = wrapper.getByText('10');
```